### PR TITLE
Microsoft Teams - add argument channel_name to mirror_investigation

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Description
Allows user to set custom channel name for mirrored channel.

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [ ] Code Review

## Dependencies
None

### CODE REVIEW

**Comment by Itay4** [comment]
@kirbles19 please review the yml and the md files

File: Integrations/MicrosoftTeams/MicrosoftTeams.yml [hunk 1]
```diff
@@ -160,7 +172,6 @@ script:
   runonce: false
   script: '-'
   type: python
```
Comment (yaakovi) [inline Integrations/MicrosoftTeams/MicrosoftTeams.yml]: why the subtype is removed?

**Comment by Itay4** [comment]
@richardbluestone @kirbles19 whats our status here?

**Comment by richardbluestone** [comment]
I'm not sure what needs to be reviewed as we have already added the channel_name arguments in the the mirror-investigation command:  https://support.demisto.com/hc/en-us/articles/360034095994-Microsoft-Teams#h_703f92ca-5161-4501-ada4-b47fe63a4d79

**Pull request merged**